cmd/argocd-application-controller/commands/argocd_application_controller.go

```diff
@@ -174,6 +176,21 @@ func NewCommand() *cobra.Command {
 			errors.CheckError(err)
 			cacheutil.CollectMetrics(redisClient, appController.GetMetricsServer())
 
+			// run pprof server
+			if pprof.IsEnabled() {
+				pprofSrv, err := pprof.NewPprofServer()
+				if err != nil {
+					log.Error(err, "failed to create pprof handler")
+					os.Exit(1)
```

Comment:
```suggestion
					log.Fatal(err, "failed to create pprof handler")
```

There's a short-hand for this